provider/resource_robot_account_test.go

```diff
@@ -25,8 +25,30 @@ func TestAccRobotBasic(t *testing.T) {
 					testAccCheckResourceExists(harborRobotAccount),
 					resource.TestCheckResourceAttr(
 						harborRobotAccount, "name", "test_robot_account"),
+					// resource.TestCheckResourceAttr(
+					// 	harborRobotAccount, "action", "push"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccRobotMultipleAction(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckRobotDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckRobotMultipleAction(),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckResourceExists("harbor_project.main"),
+
+					testAccCheckResourceExists(harborRobotAccount),
 					resource.TestCheckResourceAttr(
-						harborRobotAccount, "action", "push"),
+						harborRobotAccount, "name", "test_robot_account"),
+					// resource.TestCheckResourceAttr(
```

Comment:
shouldnt this be removed ?

Reply:
hmmmm I should actually fix the test so it checks for a list of attributes